iml-system-test-utils/src/vagrant.rs

```diff
@@ -54,12 +54,24 @@ pub async fn up<'a>() -> Result<Command, CmdError> {
     Ok(x)
 }
 
-pub async fn destroy<'a>() -> Result<(), CmdError> {
-    let mut x = vagrant().await?;
+pub async fn destroy<'a>(config: &ClusterConfig) -> Result<(), CmdError> {
+    let nodes = config.destroy_list();
+
+    for node in &nodes {
+        let mut suspend_cmd = suspend().await?;
+        suspend_cmd.arg(node);
```

Comment:
So it looks like a network outage, right?
Something bad happen when a node is shutting down?

Reply:
Basically we are running `vagrant destroy -f` which will attempt to destroy all nodes, starting with c8 -> c1, oss2, oss1, mds2,mds1,adm,iscsi. This process will attempt to shut down each node and then destroy it along with resources one by one. This new approach will loop through each node and suspend each node. By doing it this way, we ensure everything is suspended before we attempt to destroy. This will hopefully prevent resources from getting hung up.